docs/admin/config.rst

```diff
@@ -892,14 +892,34 @@ API key for DeepL API, you can register at https://www.deepl.com/pro.html.
 MT_GOOGLE_KEY
 -------------
 
-API key for Google Translate API, you can register at https://cloud.google.com/translate/docs
+API key for Google Translate API v3, you can register at https://cloud.google.com/translate/docs
```

Comment:
I think this one is actually for v2, isn't it?

Reply:
Yes, this is mistake.